config/commands.json

```diff
@@ -898,6 +898,22 @@
     }
   },
   
+  "!why": {
+    "usage": "",
+    "description": "Why should I use LBRY?",
+    "operation": "send",
+    "bundle": {
+      "url": "",
+      "title": "",
+      "description": "LBRY is a new protocol which allows anyone to build apps that interact with digital content on the LBRY network. Apps built on the protocol allow creators to upload their work to the LBRY network of hosts (like BitTorrent), and set a price per stream or download (like iTunes) or give it away for free (like YouTube without ads). It benefits the consumer, because there's no ads, and you can often buy things cheaper on LBRY, and it benefits the creator, because there's no risk of demonetization, essentially no fees, and it's a lot harder to censor. You can find out more on the [FAQ](https://lbry.com/faq), or ask anyone here for their experience!",
+      "color": 7976557,
+      "author": {
+        "name": "Why should I use LBRY?",
+        "url": "https://lbry.com/faq",
```

Comment:
Is this the correct/intended URL?

Reply:
It's the one I feel is best because they can then go straight to the question and reasons to use LBRY for their use case, but if there's a better URL I can change that

Reply:
I can agree with ICT, unless we want to point them to the https://lbry.com/what page. @kauffj